content/04.results.md

```diff
@@ -2,42 +2,54 @@
 
 ### Generative Model Using Randomly Sampled Label Functions
 ![
-Grid of AUROC scores for each generative model trained on randomly sampled label functions.
-The rows depict the relationship each model is trying to predict and the columns are the edge type specific sources from which each label function is sampled.
-The right most column consists of pooling every relationship specific label function and proceeding as above.
-](https://raw.githubusercontent.com/danich1/snorkeling/ee638b4e45717a86f54a2744a813baaa90bc6b84/figures/label_sampling_experiment/transfer_test_set_auroc.png){#fig:auroc_gen_model_performance}
+Plot of AUROC scores for each generative model trained on randomly sampled label functions.
```

Comment:
As with other figures - give me a short summary of your takeaway in the first line of the legend.